examples/storage-examples/ceph-examples/MYSQL.md

```diff
@@ -0,0 +1,124 @@
+Setting Up and Validating Containerized MySQL
+=============================================
+
+First, OSE's Security Context Constraints (SSC) need to be defined such that the *seLinuxContext* and *runAsUser* values are set to "RunAsAny". Selinux is still enabled/enforcing on the OSE-master and OSE-nodes, as decribed in the [OSE environment readme](ENV.md).
+
+The mysql example uses the official mysql image found [here](https://hub.docker.com/_/mysql/). First, pull down the image on *each* OSE-nodes:
```

Comment:
this should use the openshift provided mysql image instead, which doesn't require any security changes to work....
https://github.com/openshift/mysql